app/models/resource.py

```diff
@@ -71,6 +71,7 @@ class Resource(db.Model):
     __tablename__ = 'resources'
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(64), index=True)
+    suggestions = db.relationship('Suggestion', backref='resource', uselist=True)
```

Comment:
(very) nit: can you move this under `option_descriptors`, please? I think it'll preserve the order of vitality of fields to a resource :)